src/routes/single-page-app.js

```diff
@@ -10,6 +10,27 @@ const pageRoutes = require("../data/routes");
 
 const singlePageAppRouter = Router();
 
+singlePageAppRouter.get(
+  "^/locales/:lang([a-zA-Z]{2})/translation.json",
```

Comment:
This feels like unnecessary code for returning static files. It looks like we can use express.static() to serve these files from the client/locales directory. https://expressjs.com/en/starter/static-files.html

It looks like we can combine it with the [compression](http://expressjs.com/en/resources/middleware/compression.html) middleware to further improve the performance (maybe in a separate change).

Reply:
serving them as static files was creating the caching issue in the first place.